starter-code/index.html

```diff
@@ -1,32 +1,57 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <title>Spotify</title>
-</head>
-<body>
-Premium Discover Help Download
+  <head>
+    <meta charset="UTF-8">
+    <title>Spotify</title>
+    <link rel="stylesheet" href="css/main.css">
+  </head>
+  <body>
+    <header>
+      <img src="images/spotify-logo.png" class="lego-img">
+      <nav>
+        <ul>
+          <a href="#1">Premium</a>
+          <a href="#1">Discover</a>
+          <a href="#1">Help</a>
+          <a href="#1">Download</a>
+        </ul>
+      </nav>
+    </header>
+    <section>
+      <div class="cover-img">
+        <h1>Music for everyone</h1>
+          <p class="p-h1">Spotify is now free on mobile, tablet and computer.
+          Listen to the right music, wherever you are.</p>
+      </div>
+    </section>
 
-Music for everyone.
-Spotify is now free on mobile, tablet and computer.
-Listen to the right music, wherever you are.
+    <section class="section-2">
+          <h2><a target="_blank" href="https://www.spotify.com/us/">What’s on Spotify?</a></h2>
```

Comment:
Instead of wrapping the a tag in a h2 tag, try wrapping the h2 tag in an a tag.

`<a><h1>....</h1></a>`